### DISCUSSION TITLE
how long can we get the access to chat and voice preview/feature

### DISCUSSION THREAD

**Discussion by JennyJiBing** [community/community#52536] [Copilot Conversations] [answered]
### Select Topic Area

General

### Body

We submitted a request for chat and voice preview features more than two weeks ago, and upon checking the status today, we were informed that we are still on the waiting list. Please see the below screenshot. We are eager to know when we can gain access to the chat and voice preview, as we can't wait to try it out.
<img width="444" alt="evidence_waitinglist" src="https://user-images.githubusercontent.com/127969025/231203278-5a3595b9-8f5e-4866-9465-e9da8e86aaf4.png">

**Comment by Akash1134** [comment]
Hi @JennyJiBing ,

GitHub Copilot X is currently a representation of GitHub’s vision for the future rather than an available product offering of GitHub Copilot. As we continue to design, test, and build features that fall into the GitHub Copilot X vision, we are also taking the time to determine the best way to provide them to our customers.

Currently, there is no way to find out one's position on the Copilot Chat waitlist. Also note that, Copilot Chat is only available for **public preview** - meaning the waitlist is for getting yourself enrolled into the technical preview which will be available sometime in the future.

You can explore the Copilot X features we offer to preview [here](https://github.com/features/preview/copilot-x).

**Reply by JennyJiBing** [reply]
Thank you for responding. We understand that Copilot chat is currently only available for public preview. Could you kindly inform us of when we can gain access to Copilot voice technical view? Our company is eagerly waiting to utilize it and we are still on the waiting list. Thank you again!

**Comment by knittingguy** [comment]
I signed up for the Copilot Chat wait list about 2 months ago and it only took them about a week to approve.  However, I have signed up for the Copilot Voice for 2 weeks now and am still waiting.

**Reply by MMALI3287** [reply]
### I signed up for the chat and got it almost immediately, but I haven't gotten the voice chat yet, even though months have passed.

**Comment by Trubador** [comment]
I am also eagerly waiting to get access since it will speed up my development workflow a lot and I have been on the waitlist for months now.

**Comment by davecheney** [comment, marked as answer]
Chat is available for all VSCode and Visual Studio users. Jetbrains remains waitlisted due to high levels of uptake. 

The github next team have announced, I think last year, that voice chat has been discontinued.

**Comment by knittingguy** [comment]
The closest thing would be the VS Code Speech extension.  VS Code Speech extension has a pre-requisite of VS Copilot Chat.  With the new release, you can now install the language packs after installing the VS Code Speech extension.
